src/patternfly/components/DataToolbar/data-toolbar.scss

```diff
@@ -36,8 +36,8 @@ $pf-c-data-toolbar--spacer-map: build-spacer-map("none", "sm", "md", "lg");
   --pf-c-data-toolbar__expandable-content--m-expanded--GridRowGap: var(--pf-global--gutter);
 
   // Chip container
-  --pf-c-data-toolbar__content--m-chip-container--MarginTop: calc(var(--pf-global--spacer--md) * -1);
-  --pf-c-data-toolbar__content--m-chip-container__item--MarginTop: var(--pf-global--spacer--md);
+  --pf-c-data-toolbar__group--m-chip-container--MarginTop: calc(var(--pf-global--spacer--md) * -1);
+  --pf-c-data-toolbar__group--m-chip-container--item--MarginTop: var(--pf-global--spacer--md);
```

Comment:
why wouldn't this var and the one above it be

`--pf-c-data-toolbar__expandable-content--m-chip-container--MarginTop` and `--pf-c-data-toolbar__expandable-content--m-chip-container__item--MarginTop`?

Reply:
@mcoker updated